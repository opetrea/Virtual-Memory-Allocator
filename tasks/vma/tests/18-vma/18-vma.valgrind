==91895== Memcheck, a memory error detector
==91895== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91895== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91895== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91895== Parent PID: 91894
==91895== 
==91895== 
==91895== HEAP SUMMARY:
==91895==     in use at exit: 0 bytes in 0 blocks
==91895==   total heap usage: 1,008 allocs, 1,008 frees, 35,528 bytes allocated
==91895== 
==91895== All heap blocks were freed -- no leaks are possible
==91895== 
==91895== For lists of detected and suppressed errors, rerun with: -s
==91895== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
