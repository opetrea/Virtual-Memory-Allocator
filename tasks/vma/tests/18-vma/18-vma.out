Total memory: 0x64 bytes
Free memory: 0x0 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 100

Block 1 begin
Zone: 0x0 - 0x64
Miniblock 1:		0x0		-		0x1		| RW-
Miniblock 2:		0x1		-		0x2		| RW-
Miniblock 3:		0x2		-		0x3		| RW-
Miniblock 4:		0x3		-		0x4		| RW-
Miniblock 5:		0x4		-		0x5		| RW-
Miniblock 6:		0x5		-		0x6		| RW-
Miniblock 7:		0x6		-		0x7		| RW-
Miniblock 8:		0x7		-		0x8		| RW-
Miniblock 9:		0x8		-		0x9		| RW-
Miniblock 10:		0x9		-		0xA		| RW-
Miniblock 11:		0xA		-		0xB		| RW-
Miniblock 12:		0xB		-		0xC		| RW-
Miniblock 13:		0xC		-		0xD		| RW-
Miniblock 14:		0xD		-		0xE		| RW-
Miniblock 15:		0xE		-		0xF		| RW-
Miniblock 16:		0xF		-		0x10		| RW-
Miniblock 17:		0x10		-		0x11		| RW-
Miniblock 18:		0x11		-		0x12		| RW-
Miniblock 19:		0x12		-		0x13		| RW-
Miniblock 20:		0x13		-		0x14		| RW-
Miniblock 21:		0x14		-		0x15		| RW-
Miniblock 22:		0x15		-		0x16		| RW-
Miniblock 23:		0x16		-		0x17		| RW-
Miniblock 24:		0x17		-		0x18		| RW-
Miniblock 25:		0x18		-		0x19		| RW-
Miniblock 26:		0x19		-		0x1A		| RW-
Miniblock 27:		0x1A		-		0x1B		| RW-
Miniblock 28:		0x1B		-		0x1C		| RW-
Miniblock 29:		0x1C		-		0x1D		| RW-
Miniblock 30:		0x1D		-		0x1E		| RW-
Miniblock 31:		0x1E		-		0x1F		| RW-
Miniblock 32:		0x1F		-		0x20		| RW-
Miniblock 33:		0x20		-		0x21		| RW-
Miniblock 34:		0x21		-		0x22		| RW-
Miniblock 35:		0x22		-		0x23		| RW-
Miniblock 36:		0x23		-		0x24		| RW-
Miniblock 37:		0x24		-		0x25		| RW-
Miniblock 38:		0x25		-		0x26		| RW-
Miniblock 39:		0x26		-		0x27		| RW-
Miniblock 40:		0x27		-		0x28		| RW-
Miniblock 41:		0x28		-		0x29		| RW-
Miniblock 42:		0x29		-		0x2A		| RW-
Miniblock 43:		0x2A		-		0x2B		| RW-
Miniblock 44:		0x2B		-		0x2C		| RW-
Miniblock 45:		0x2C		-		0x2D		| RW-
Miniblock 46:		0x2D		-		0x2E		| RW-
Miniblock 47:		0x2E		-		0x2F		| RW-
Miniblock 48:		0x2F		-		0x30		| RW-
Miniblock 49:		0x30		-		0x31		| RW-
Miniblock 50:		0x31		-		0x32		| RW-
Miniblock 51:		0x32		-		0x33		| RW-
Miniblock 52:		0x33		-		0x34		| RW-
Miniblock 53:		0x34		-		0x35		| RW-
Miniblock 54:		0x35		-		0x36		| RW-
Miniblock 55:		0x36		-		0x37		| RW-
Miniblock 56:		0x37		-		0x38		| RW-
Miniblock 57:		0x38		-		0x39		| RW-
Miniblock 58:		0x39		-		0x3A		| RW-
Miniblock 59:		0x3A		-		0x3B		| RW-
Miniblock 60:		0x3B		-		0x3C		| RW-
Miniblock 61:		0x3C		-		0x3D		| RW-
Miniblock 62:		0x3D		-		0x3E		| RW-
Miniblock 63:		0x3E		-		0x3F		| RW-
Miniblock 64:		0x3F		-		0x40		| RW-
Miniblock 65:		0x40		-		0x41		| RW-
Miniblock 66:		0x41		-		0x42		| RW-
Miniblock 67:		0x42		-		0x43		| RW-
Miniblock 68:		0x43		-		0x44		| RW-
Miniblock 69:		0x44		-		0x45		| RW-
Miniblock 70:		0x45		-		0x46		| RW-
Miniblock 71:		0x46		-		0x47		| RW-
Miniblock 72:		0x47		-		0x48		| RW-
Miniblock 73:		0x48		-		0x49		| RW-
Miniblock 74:		0x49		-		0x4A		| RW-
Miniblock 75:		0x4A		-		0x4B		| RW-
Miniblock 76:		0x4B		-		0x4C		| RW-
Miniblock 77:		0x4C		-		0x4D		| RW-
Miniblock 78:		0x4D		-		0x4E		| RW-
Miniblock 79:		0x4E		-		0x4F		| RW-
Miniblock 80:		0x4F		-		0x50		| RW-
Miniblock 81:		0x50		-		0x51		| RW-
Miniblock 82:		0x51		-		0x52		| RW-
Miniblock 83:		0x52		-		0x53		| RW-
Miniblock 84:		0x53		-		0x54		| RW-
Miniblock 85:		0x54		-		0x55		| RW-
Miniblock 86:		0x55		-		0x56		| RW-
Miniblock 87:		0x56		-		0x57		| RW-
Miniblock 88:		0x57		-		0x58		| RW-
Miniblock 89:		0x58		-		0x59		| RW-
Miniblock 90:		0x59		-		0x5A		| RW-
Miniblock 91:		0x5A		-		0x5B		| RW-
Miniblock 92:		0x5B		-		0x5C		| RW-
Miniblock 93:		0x5C		-		0x5D		| RW-
Miniblock 94:		0x5D		-		0x5E		| RW-
Miniblock 95:		0x5E		-		0x5F		| RW-
Miniblock 96:		0x5F		-		0x60		| RW-
Miniblock 97:		0x60		-		0x61		| RW-
Miniblock 98:		0x61		-		0x62		| RW-
Miniblock 99:		0x62		-		0x63		| RW-
Miniblock 100:		0x63		-		0x64		| RW-
Block 1 end
Total memory: 0x64 bytes
Free memory: 0x64 bytes
Number of allocated blocks: 0
Number of allocated miniblocks: 0
